spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "spec_helper"
+
+describe ReviewsController do
+  describe "POST create" do
+    context "with authenticated user" do
+      before do
+        @user = Fabricate(:user)
+        session[:user_id] = @user.id
+        @video = Fabricate(:video)
+      end
+      it "expects a logged in user" do
+        post :create, review: Fabricate.attributes_for(:review), video_id: @video.id
+        expect(session[:user_id]).not_to be(nil)
```

Comment:
You are not testing your code here, you are testing your test setup. You should only test code that is affected when you access the post route.